hpy/universal/src/debugmode.h

```diff
@@ -0,0 +1,12 @@
+#ifndef HPY_DEBUGMODE_H
+#define HPY_DEBUGMODE_H
+
+void _hpy_fatal(const char *msg, ...);
+
+#define HPY_ASSERT(condition, errargs)    \
+    do {                                  \
+        if (!(condition))                 \
+            _hpy_fatal errargs;           \
```

Comment:
we can probably use `__VA_ARGS__` to avoid the weird usage pattern in which you have to put parenthesis in the call?